runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -929,6 +931,9 @@ inlineIntrinsicIndexOf(TR::Node * node, TR::CodeGenerator * cg, bool isLatin1)
    TR::Register* offset = cg->evaluate(node->getChild(3));
    TR::Register* length = cg->gprClobberEvaluate(node->getChild(4));
 
+
+   const int8_t sizeOfVector = cg->machine()->getVRFSize();
```

Comment:
There are some failures in the builds:

```
> c++filt _Z39generateS390CompareAndBranchInstructionIaEPN2TR11InstructionEPNS0_13CodeGeneratorEN3OMR10InstOpCode8MnemonicEPNS0_4NodeEPNS0_8RegisterET_NS5_1Z10InstOpCode19S390BranchConditionEPNS0_11LabelSymbolEbbS2_PNS0_28RegisterDependencyConditionsE
TR::Instruction* generateS390CompareAndBranchInstruction<signed char>(TR::CodeGenerator*, OMR::InstOpCode::Mnemonic, TR::Node*, TR::Register*, signed char, OMR::Z::InstOpCode::S390BranchCondition, TR::LabelSymbol*, bool, bool, TR::Instruction*, TR::RegisterDependencyConditions*)
```

It looks like the template was attempted to be instantiated with `signed char` and we couldn't find a definition, because we only explicitly instantiate the `int32_t` and `int64_t` versions [1]. To fix the error I think we need to change the type of `sizeOfVector` to `const int32_t`. Similar change needed in the other function.

[1] https://github.com/eclipse/omr/blob/7d6908d059acf3e30536719a1ebaaec38116a5a1/compiler/z/codegen/S390GenerateInstructions.cpp#L3189-L3213

Reply:
Fixed in 0631f02